src/panels/lovelace/cards/hui-alarm-panel-card.js

```diff
@@ -0,0 +1,229 @@
+          <template is='dom-if' if='[[_showActionToggle(_stateObj.state)]]'>
+            <div id='ArmActions' class='actions'>
+              <template is='dom-repeat' items='[[_config.states]]'>
+                <paper-button noink raised id='[[item]]' on-click='_handleActionClick'>[[_label(item)]]</paper-button>
+              </template>
+            </div>
+          </template>
+          <template is='dom-if' if='[[!_showActionToggle(_stateObj.state)]]'>
+            <div id='DisarmActions' class='actions'>
+              <paper-button noink raised id='disarm' on-click='_handleActionClick'>[[_label('disarm')]]</paper-button>
+            </div>
+          </template>
+          <paper-input id='Input' label='Alarm Code' type='password' value='[[_value]]'></paper-input>
+          <div id='Keypad'>
+              <div id='ButtonSection1'>
+                <paper-button noink raised value='1' on-click='_handlePadClick'>1</paper-button>
+                <paper-button noink raised value='4' on-click='_handlePadClick'>4</paper-button>
+                <paper-button noink raised value='7' on-click='_handlePadClick'>7</paper-button>
+              </div>
+              <div id='ButtonSection2'>
+                <paper-button noink raised value='2' on-click='_handlePadClick'>2</paper-button>
+                <paper-button noink raised value='5' on-click='_handlePadClick'>5</paper-button>
+                <paper-button noink raised value='8' on-click='_handlePadClick'>8</paper-button>
+                <paper-button noink raised value='0' on-click='_handlePadClick'>0</paper-button>
+              </div>
+              <div id='ButtonSection3'>
+                <paper-button noink raised value='3' on-click='_handlePadClick'>3</paper-button>
+                <paper-button noink raised value='6' on-click='_handlePadClick'>6</paper-button>
+                <paper-button noink raised value='9' on-click='_handlePadClick'>9</paper-button>
+                <paper-button noink raised value='clear' on-click='_handlePadClick'>CLEAR</paper-button>
+              </div>
+        </div>
+      </template>
+      <template is='dom-if' if='[[!_stateObj]]'>
+        <div>Entity not available: [[_config.entity]]</div>
+      </template>
+    </ha-card>
+    `;
+  }
+
+  static get properties() {
+    return {
+      hass: {
+        type: Object
+      },
+      _config: Object,
+      _stateObj: {
+        type: Object,
+        computed: '_computeStateObj(hass.states, _config.entity)',
+      },
+      _value: {
+        type: String,
+        value: ''
+      },
+    };
+  }
+
+  getCardSize() {
+    return 1;
+  }
+
+  setConfig(config) {
+    if (!config || !config.entity || config.entity.split('.')[0] !== 'alarm_control_panel') throw new Error('Invalid card configuration');
+    this._config = Object.assign({ states: ['arm_away', 'arm_home'] }, config);
+    this._icons = {
+      armed_away: 'mdi:security-lock',
```

Comment:
all icons need to be `hass:` instead of `mdi:`. After changing, restart your development script and it will generate new hass icon set with those included.